setup.py

```diff
@@ -7,7 +7,7 @@
 
 setup(
     name='python-documentcloud',
-    version='3.4.0',
+    version='3.4.1',
```

Comment:
should be 3.5.0 - I don't think I have been super consistent with this, but the first number is for backward incompatible changes, second number is for backward compatible new features, and last number is for backward compatible bug fixes.